src/components/Header/HeaderContainer.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import {
+  Form,
+  TextInput,
+  RowContainer,
+  IconContainer,
+} from './HeaderPresenter';
+
+const HeaderContainer = () => {
+  const [text, setText] = useState('');
+
+  const handleSubmitButtonClick = () => {
+    // 일단은 아무일도 안 일어나고 초기화만
+    setText('');
+    window.alert('미구현입니다~');
+  };
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    handleSubmitButtonClick();
+  };
+  const onChange = (e) => {
+    const nextForm = e.target.value;
+    setText(nextForm);
+  };
+
+  return (
+    <RowContainer>
+      <img src="img/logo.png" width="103px" height="29px" alt="logo"></img>
+      <Form onSubmit={handleSubmit}>
+        <TextInput
+          type="text"
+          placeholder="검색"
+          onChange={onChange}
+          value={text}
+        />
+      </Form>
+      <IconContainer>
+        <img className="icon" src="img/home.png" width="22px" alt="home"></img>
```

Comment:
```suggestion
        <img className="icon" src="img/home.png" width="22px" alt="home"/>
```
굳이 수정하지 않아도 되는 사소한 부분이지만... 이렇게 작성하시면 조금이라도 깔끔해 보이지 않을까 싶습니다...!